src/GraphBLAS-sharp/Algorithms/SSSP.fs

```diff
@@ -0,0 +1,15 @@
+namespace GraphBLAS.FSharp.Algorithms
+
+open GraphBLAS.FSharp.Predefined
+open GraphBLAS.FSharp
+
+[<AutoOpen>]
+module SSSP =
```

Comment:
I think that SSSP should be defined using R^\infinite semiring. Or we assume that the distance between two nodes cannot be 0? If we use R as a domain, then distance can be equals to 0. Thus we should use R^\infinite or R+.